### PR TITLE
Techdebt/add spaces inside parens

### PR DESCRIPTION
**Pull request by sarahsturgeon**

### CODE REVIEW

File: lua/cfcfactions/core/cl/cl_utilities.lua [hunk 1]
```diff
@@ -19,28 +19,28 @@ function cfcFactions:ResizeChildrenEqually(parent, modifer)
...
     if parent ~= nil and parent:HasChildren() then
-        for k, babypanel in pairs(parent:GetChildren()) do
+        for k, babypanel in pairs( parent:GetChildren() ) do
             
             if babypanel:GetClassName() == "Label" then
-                --print(string.format("Looping on: %s with size of %s",babypanel:GetText(), #babypanel:GetText()))
+                --print(string.format("Looping on: %s with size of %s",babypanel:GetText(), #babypanel:GetText( )) )
                 if #babypanel:GetText() > MaxSize then
-                    MaxSize = ((#babypanel:GetText()) * mulmod)
+                    MaxSize = ( (#babypanel:GetText() ) * mulmod)
```
Comment (sarahsturgeon) [inline lua/cfcfactions/core/cl/cl_utilities.lua]:
```suggestion
                    MaxSize = ( ( #babypanel:GetText() ) * mulmod )
```

File: lua/cfcfactions/core/cl/dermas/cl_alertsderma.lua [hunk 2]
```diff
@@ -19,14 +19,14 @@ local function printyprint(number)
...
+local function drawAlertsTable( main_panel )
     if lastCount == -1 then
         MsgN("Current number of alerts: #" .. #cfcFactions.Alerts)
     end
     if (#cfcFactions.Alerts ~= lastCount) or (lastCount == -1) then
         main_panel.logview:Clear()
-        for k, logs in pairs(cfcFactions.Alerts) do
-            main_panel.logview:AddLine(logs.Time, logs.Message,printyprint(logs.Type))
+        for k, logs in pairs( cfcFactions.Alerts ) do
+            main_panel.logview:AddLine(logs.Time, logs.Message,printyprint( logs.Type) )
```
Comment (sarahsturgeon) [inline lua/cfcfactions/core/cl/dermas/cl_alertsderma.lua]:
```suggestion
            main_panel.logview:AddLine( logs.Time, logs.Message,printyprint( logs.Type) )
```

File: lua/cfcfactions/core/cl/dermas/cl_creditsderma.lua [hunk 3]
```diff
@@ -7,16 +7,16 @@ cfcFactions:RegisterDermaMenu("View Credits", Panel, 10)
...
     local Credits = cfcFactions.Credits.GenerateDeveloperNames()
     local TextBlock = ""
-    for KEY, Developer in pairs(Credits) do
+    for KEY, Developer in pairs( Credits ) do
     	TextBlock = TextBlock .. (Developer .. "(" .. KEY .. ")\n")
     end
-    self.TextBox:SetText(TextBlock)
-    self.TextBox:SetColor(Color(0,0,0,255))
+    self.TextBox:SetText( TextBlock )
+    self.TextBox:SetColor( Color(0,0,0,255) )
```
Comment (sarahsturgeon) [inline lua/cfcfactions/core/cl/dermas/cl_creditsderma.lua]:
```suggestion
    self.TextBox:SetColor( Color( 0, 0, 0, 255 ) )
```

File: lua/cfcfactions/core/cl/dermas/cl_factionsderma.lua [hunk 4]
```diff
@@ -16,93 +16,93 @@ function Panel:Init()
...
 
     self.BottomPanel = vgui.Create("DPanel", self.MainContainer)
-    self.BottomPanel:Dock(BOTTOM)
-    self.BottomPanel:SetBackgroundColor(Color(0,0,0,0))
-    self.BottomPanel:SetWide(self:GetWide())
-    self.BottomPanel:SetTall(50)
-    self.BottomPanel:InvalidateParent(true)
-    self.BottomPanel:SetBackgroundColor(ColorAlpha(cfg.ColorSchemes.BackgroundPanel,255))
+    self.BottomPanel:Dock( BOTTOM )
+    self.BottomPanel:SetBackgroundColor( Color(0,0,0,0) )
```
Comment (sarahsturgeon) [inline lua/cfcfactions/core/cl/dermas/cl_factionsderma.lua]:
```suggestion
    self.BottomPanel:SetBackgroundColor( Color(0, 0, 0, 0) )
```

File: lua/cfcfactions/core/cl/dermas/cl_factionsderma.lua [hunk 5]
```diff
@@ -16,93 +16,93 @@ function Panel:Init()
...
+    self.BottomPanel:SetWide( self:GetWide() )
+    self.BottomPanel:SetTall( 50 )
+    self.BottomPanel:InvalidateParent( true )
+    self.BottomPanel:SetBackgroundColor( ColorAlpha(cfg.ColorSchemes.BackgroundPanel,255) )
```
Comment (sarahsturgeon) [inline lua/cfcfactions/core/cl/dermas/cl_factionsderma.lua]:
```suggestion
    self.BottomPanel:SetBackgroundColor( ColorAlpha( cfg.ColorSchemes.BackgroundPanel, 255 ) )
```

File: lua/cfcfactions/core/cl/dermas/cl_factionsderma.lua [hunk 6]
```diff
@@ -16,93 +16,93 @@ function Panel:Init()
...
 
     self.BottomContainerTop = vgui.Create("DPanel", self.BottomPanel)
-    self.BottomContainerTop:Dock(TOP)
-    self.BottomContainerTop:SetBackgroundColor(ColorAlpha(cfg.ColorSchemes.BackgroundPanel,255))
+    self.BottomContainerTop:Dock( TOP )
+    self.BottomContainerTop:SetBackgroundColor( ColorAlpha(cfg.ColorSchemes.BackgroundPanel,255) )
```
Comment (sarahsturgeon) [inline lua/cfcfactions/core/cl/dermas/cl_factionsderma.lua]:
```suggestion
    self.BottomContainerTop:SetBackgroundColor( ColorAlpha( cfg.ColorSchemes.BackgroundPanel, 255 ) )
```

File: lua/cfcfactions/core/cl/dermas/cl_factionsderma.lua [hunk 7]
```diff
@@ -16,93 +16,93 @@ function Panel:Init()
...
 
     self.BottomContainerBottom = vgui.Create("DPanel", self.BottomPanel)
-    self.BottomContainerBottom:Dock(BOTTOM)
-    self.BottomContainerBottom:SetBackgroundColor(ColorAlpha(cfg.ColorSchemes.BackgroundPanel,255))
+    self.BottomContainerBottom:Dock( BOTTOM )
+    self.BottomContainerBottom:SetBackgroundColor( ColorAlpha(cfg.ColorSchemes.BackgroundPanel,255) )
```
Comment (sarahsturgeon) [inline lua/cfcfactions/core/cl/dermas/cl_factionsderma.lua]:
```suggestion
    self.BottomContainerBottom:SetBackgroundColor( ColorAlpha( cfg.ColorSchemes.BackgroundPanel, 255 ) )
```

File: lua/cfcfactions/core/cl/dermas/cl_factionsderma.lua [hunk 8]
```diff
@@ -16,93 +16,93 @@ function Panel:Init()
...
 
     --Bottom Panel - Contains buttons to interact with factions
     self.BottomButtonsControlPanel = vgui.Create("DPanel", self.BottomContainerBottom)
-    self.BottomButtonsControlPanel:SetBackgroundColor(ColorAlpha(cfg.ColorSchemes.BackgroundPanel,255))
-    self.BottomButtonsControlPanel:Dock(BOTTOM)
+    self.BottomButtonsControlPanel:SetBackgroundColor( ColorAlpha(cfg.ColorSchemes.BackgroundPanel,255) )
```
Comment (sarahsturgeon) [inline lua/cfcfactions/core/cl/dermas/cl_factionsderma.lua]:
```suggestion
    self.BottomButtonsControlPanel:SetBackgroundColor( ColorAlpha( cfg.ColorSchemes.BackgroundPanel, 255 ) )
```

File: lua/cfcfactions/core/cl/dermas/cl_factionsderma.lua [hunk 9]
```diff
@@ -16,93 +16,93 @@ function Panel:Init()
...
+    self.BottomButtonsControlPanel:Dock( BOTTOM )
 
     --Factions View buttons (Changing pages)
     self.ButtonsContainer = vgui.Create("DPanel", self.BottomContainerTop)
-    self.ButtonsContainer:Dock(RIGHT)
-    self.ButtonsContainer:InvalidateParent(true)
-    self.ButtonsContainer:SetBackgroundColor(ColorAlpha(cfg.ColorSchemes.BackgroundPanel,255))
+    self.ButtonsContainer:Dock( RIGHT )
+    self.ButtonsContainer:InvalidateParent( true )
+    self.ButtonsContainer:SetBackgroundColor( ColorAlpha(cfg.ColorSchemes.BackgroundPanel,255) )
```
Comment (sarahsturgeon) [inline lua/cfcfactions/core/cl/dermas/cl_factionsderma.lua]:
```suggestion
    self.ButtonsContainer:SetBackgroundColor( ColorAlpha( cfg.ColorSchemes.BackgroundPanel, 255 ) )
```

File: lua/cfcfactions/core/cl/dermas/cl_factionsderma.lua [hunk 10]
```diff
@@ -16,93 +16,93 @@ function Panel:Init()
...
-    -- self.ButtonsContainer:SetWide(210)
+    -- self.ButtonsContainer:SetWide( 210 )
 
     self.ButtonsPanelLeft = vgui.Create("DPanel", self.ButtonsContainer)
-    self.ButtonsPanelLeft:Dock(LEFT)
-    self.ButtonsPanelLeft:InvalidateParent(true)
-    self.ButtonsPanelLeft:SetBackgroundColor(ColorAlpha(cfg.ColorSchemes.BackgroundPanel,255))
+    self.ButtonsPanelLeft:Dock( LEFT )
+    self.ButtonsPanelLeft:InvalidateParent( true )
+    self.ButtonsPanelLeft:SetBackgroundColor( ColorAlpha(cfg.ColorSchemes.BackgroundPanel,255) )
```
Comment (sarahsturgeon) [inline lua/cfcfactions/core/cl/dermas/cl_factionsderma.lua]:
```suggestion
    self.ButtonsPanelLeft:SetBackgroundColor( ColorAlpha( cfg.ColorSchemes.BackgroundPanel, 255 ) )
```

File: lua/cfcfactions/core/cl/dermas/cl_factionsderma.lua [hunk 11]
```diff
@@ -16,93 +16,93 @@ function Panel:Init()
...
     self.ButtonsPanelRight = vgui.Create("DPanel", self.ButtonsContainer)
-    self.ButtonsPanelRight:Dock(RIGHT)
-    self.ButtonsPanelRight:InvalidateParent(true)
-    self.ButtonsPanelRight:SetBackgroundColor(ColorAlpha(cfg.ColorSchemes.BackgroundPanel,255))
+    self.ButtonsPanelRight:Dock( RIGHT )
+    self.ButtonsPanelRight:InvalidateParent( true )
+    self.ButtonsPanelRight:SetBackgroundColor( ColorAlpha(cfg.ColorSchemes.BackgroundPanel,255) )
```
Comment (sarahsturgeon) [inline lua/cfcfactions/core/cl/dermas/cl_factionsderma.lua]:
```suggestion
    self.ButtonsPanelRight:SetBackgroundColor( ColorAlpha( cfg.ColorSchemes.BackgroundPanel, 255 ) )
```

File: lua/cfcfactions/core/cl/dermas/cl_factionsderma.lua [hunk 12]
```diff
@@ -16,93 +16,93 @@ function Panel:Init()
...
-    self.LastPage:SetText(">>")
-    self.LastPage:Dock(RIGHT)
+    self.LastPage:SetText( ">>" )
+    self.LastPage:Dock( RIGHT )
 
-    self.ButtonsPanelLeft:SetWide(self.FirstPage:GetWide() + self.NextPage:GetWide())
-    self.ButtonsPanelRight:SetWide(self.NextPage:GetWide() + self.LastPage:GetWide())
-    --self.ButtonsContainer:SetWide(self.FirstPage:GetWide() + self.PreviousPage:GetWide() + self.NextPage:GetWide() + self.LastPage:GetWide())
-    cfcFactions:ResizeParentFromChildren(self.ButtonsContainer)
+    self.ButtonsPanelLeft:SetWide( self.FirstPage:GetWide( ) + self.NextPage:GetWide( ) )
```
Comment (sarahsturgeon) [inline lua/cfcfactions/core/cl/dermas/cl_factionsderma.lua]:
```suggestion
    self.ButtonsPanelLeft:SetWide( self.FirstPage:GetWide() + self.NextPage:GetWide() )
```

File: lua/cfcfactions/core/cl/dermas/cl_factionsderma.lua [hunk 13]
```diff
@@ -16,93 +16,93 @@ function Panel:Init()
...
+    self.ButtonsPanelRight:SetWide( self.NextPage:GetWide( ) + self.LastPage:GetWide( ) )
```
Comment (sarahsturgeon) [inline lua/cfcfactions/core/cl/dermas/cl_factionsderma.lua]:
```suggestion
    self.ButtonsPanelRight:SetWide( self.NextPage:GetWide() + self.LastPage:GetWide() )
```

File: lua/cfcfactions/core/cl/dermas/cl_mainderma.lua [hunk 14]
```diff
@@ -81,85 +81,85 @@ end
 
 function Panel:Init()
     self:SetSize( math.Clamp( 1024, 0, ScrW() ), math.Clamp( 800, 0, ScrH() ) )
-    self:SetPos( ( (ScrW() / 2) - (self:GetWide() / 2) ), ((ScrH() / 2) - (self:GetTall() / 2) ) )
+    self:SetPos( ( ( ScrW( ) / 2) - ( self:GetWide( ) / 2) ), ( (ScrH( ) / 2) - ( self:GetTall( ) / 2) ) )
```
Comment (sarahsturgeon) [inline lua/cfcfactions/core/cl/dermas/cl_mainderma.lua]:
```suggestion
    self:SetPos( ( ( ScrW( ) / 2 ) - ( self:GetWide() / 2 ) ), ( ( ScrH() / 2 ) - ( self:GetTall() / 2 ) ) )
```

File: lua/cfcfactions/core/cl/dermas/cl_mainderma.lua [hunk 15]
```diff
@@ -81,85 +81,85 @@ end
...
-    --  cfcFactions:RegisterDermaMenu(string)
+    --  cfcFactions:RegisterDermaMenu( string )
     --      cfcFactions.Dermas
     self.menubar = vgui.Create("DPanel", self)
     self.menubar:DockMargin(0, 45, 0, 0)
-    self.menubar:Dock(TOP)
-    self.menubar:SetSize(self:GetWide() - 0.1, self:GetTall() - 745)
-    self.menubar:SetBackgroundColor(cfg.ColorSchemes.BackgroundPanel)
+    self.menubar:Dock( TOP )
+    self.menubar:SetSize( self:GetWide( ) - 0.1, self:GetTall() - 745)
```
Comment (sarahsturgeon) [inline lua/cfcfactions/core/cl/dermas/cl_mainderma.lua]:
```suggestion
    self.menubar:SetSize( self:GetWide() - 0.1, self:GetTall() - 745 )
```

File: lua/cfcfactions/core/cl/dermas/cl_mainderma.lua [hunk 16]
```diff
@@ -81,85 +81,85 @@ end
...
+    self:SetupMenubars( self.menubar )
 
     --Main self.container
     self.container = vgui.Create("DPanel", self)
     self.container:DockMargin(0, 0, 0, 0)
-    self.container:Dock(TOP)
-    self.container:SetSize(self:GetWide() - 20, self:GetTall() - 150)
-    self.container:SetPos((self:GetWide() / 2) - (self.container:GetWide() / 2), 120)
+    self.container:Dock( TOP )
+    self.container:SetSize( self:GetWide( ) - 20, self:GetTall() - 150)
```
Comment (sarahsturgeon) [inline lua/cfcfactions/core/cl/dermas/cl_mainderma.lua]:
```suggestion
    self.container:SetSize( self:GetWide() - 20, self:GetTall() - 150 )
```

File: lua/cfcfactions/core/cl/dermas/cl_mainderma.lua [hunk 17]
```diff
@@ -81,85 +81,85 @@ end
...
+    self.container:SetPos( (self:GetWide( ) / 2) - ( self.container:GetWide( ) / 2), 120)
```
Comment (sarahsturgeon) [inline lua/cfcfactions/core/cl/dermas/cl_mainderma.lua]:
```suggestion
    self.container:SetPos( ( self:GetWide() / 2 ) - ( self.container:GetWide() / 2 ), 120 )
```

File: lua/cfcfactions/core/cl/dermas/cl_mainderma.lua [hunk 18]
```diff
@@ -81,85 +81,85 @@ end
...
     self.container:SetBackgroundColor(Color(0, 0, 0, 0))
 
     --Status Bar
     self.statusbar = vgui.Create("DPanel", self)
     self.statusbar:DockMargin(0, 0, 0, 0)
-    self.statusbar:Dock(BOTTOM)
-    self.statusbar:SetSize(self:GetWide(), self:GetTall()-750)
+    self.statusbar:Dock( BOTTOM )
+    self.statusbar:SetSize( self:GetWide( ), self:GetTall( )-750 )
```
Comment (sarahsturgeon) [inline lua/cfcfactions/core/cl/dermas/cl_mainderma.lua]:
```suggestion
    self.statusbar:SetSize( self:GetWide(), self:GetTall() - 750 )
```

File: lua/cfcfactions/core/cl/dermas/cl_mainderma.lua [hunk 19]
```diff
@@ -81,85 +81,85 @@ end
...
     self.statusbar:SetBackgroundColor(Color(0, 0, 0, 0))
 
     --alertbox
     if self.alertPanel == nil then
         self.alertPanel = vgui.Create('DPanel', self.container)
-        self.alertPanel:Dock(TOP)
-        self.alertPanel:SetSize(self.container:GetWide(), 55)
+        self.alertPanel:Dock( TOP )
+        self.alertPanel:SetSize( self.container:GetWide( ), 55)
```
Comment (sarahsturgeon) [inline lua/cfcfactions/core/cl/dermas/cl_mainderma.lua]:
```suggestion
        self.alertPanel:SetSize( self.container:GetWide(), 55 )
```

File: lua/cfcfactions/core/cl/dermas/cl_mainderma.lua [hunk 20]
```diff
@@ -81,85 +81,85 @@ end
...
     end
 
     --sub_self.container
     self.mainview = vgui.Create("DPanel", self.container)
     self.mainview:DockMargin(15, 15, 15, 15)
-    self.mainview:Dock(TOP)
-    self.mainview:SetSize(self:GetWide() - 20, self:GetTall() - 230)
-    self.mainview:SetPos((self:GetWide() / 2) - (self.mainview:GetWide() / 2), 120)
+    self.mainview:Dock( TOP )
+    self.mainview:SetSize( self:GetWide( ) - 20, self:GetTall() - 230)
```
Comment (sarahsturgeon) [inline lua/cfcfactions/core/cl/dermas/cl_mainderma.lua]:
```suggestion
    self.mainview:SetSize( self:GetWide() - 20, self:GetTall() - 230 )
```

File: lua/cfcfactions/core/cl/dermas/cl_mainderma.lua [hunk 21]
```diff
@@ -81,85 +81,85 @@ end
...
+    self.mainview:SetPos( (self:GetWide( ) / 2) - ( self.mainview:GetWide( ) / 2), 120)
```
Comment (sarahsturgeon) [inline lua/cfcfactions/core/cl/dermas/cl_mainderma.lua]:
```suggestion
    self.mainview:SetPos( ( self:GetWide() / 2 ) - ( self.mainview:GetWide() / 2 ), 120 )
```

File: lua/cfcfactions/core/cl/dermas/cl_mainderma.lua [hunk 22]
```diff
@@ -81,85 +81,85 @@ end
...
 function Panel:Paint(w, h)
-        Derma_DrawBackgroundBlur(self)
+        Derma_DrawBackgroundBlur( self )
         draw.RoundedBox( 0, 0, 0, w, h, Color( 55, 55, 55, 220 ) )
         surface.SetDrawColor(Color(0, 0, 0, 255))
         surface.DrawOutlinedRect(0, 0, w, h)
 
         --surface.DrawOutlinedRect(0, 0, w, h)
-        draw.SimpleText(string.format(cfg.DermaHeaderTitle, LocalPlayer():Nick()), "CFC_Special", 5, 5, cfg.ColorSchemes.HeaderText) 
+        draw.SimpleText(string.format(cfg.DermaHeaderTitle, LocalPlayer( ):Nick() ), "CFC_Special", 5, 5, cfg.ColorSchemes.HeaderText)
```
Comment (sarahsturgeon) [inline lua/cfcfactions/core/cl/dermas/cl_mainderma.lua]:
```suggestion
        draw.SimpleText( string.format( cfg.DermaHeaderTitle, LocalPlayer():Nick() ), "CFC_Special", 5, 5, cfg.ColorSchemes.HeaderText ) 
```

File: lua/cfcfactions/core/cl/dermas/cl_mainderma.lua [hunk 23]
```diff
@@ -170,22 +170,22 @@ function Panel:ClearAlerts()
...
+function Panel:SetMainView( panel )
     --fetch children first
-    for k, v in pairs(self.mainview:GetChildren()) do
+    for k, v in pairs( self.mainview:GetChildren() ) do
         v:Clear()
     end
 
     local mview = vgui.CreateFromTable(panel, self.mainview, nil)
-    mview:SetSize(self.mainview:GetWide(), self.mainview:GetTall())
+    mview:SetSize( self.mainview:GetWide( ), self.mainview:GetTall( ) )
```
Comment (sarahsturgeon) [inline lua/cfcfactions/core/cl/dermas/cl_mainderma.lua]:
```suggestion
    mview:SetSize( self.mainview:GetWide(), self.mainview:GetTall() )
```

File: lua/cfcfactions/core/cl/dermas/cl_mainderma.lua [hunk 24]
```diff
@@ -196,20 +196,20 @@ function Panel:CreateAlert(msg, type)
...
-        Alert.ErrMsg:SetFont("CFC_Alert")
+        Alert.ErrMsg:SetFont( "CFC_Alert" )
     end
 
-    Alert.ErrMsg:SetColor(cfg.MsgType[type])
-    Alert.ErrMsg:SetSize(Alert:GetWide(), Alert:GetTall())
-    Alert:Dock(FILL)
-    surface.PlaySound("buttons/button15.wav")
+    Alert.ErrMsg:SetColor( cfg.MsgType[type] )
+    Alert.ErrMsg:SetSize( Alert:GetWide( ), Alert:GetTall( ) )
```
Comment (sarahsturgeon) [inline lua/cfcfactions/core/cl/dermas/cl_mainderma.lua]:
```suggestion
    Alert.ErrMsg:SetSize( Alert:GetWide(), Alert:GetTall() )
```

File: lua/cfcfactions/core/cl/dermas/cl_newsderma.lua [hunk 25]
```diff
@@ -6,16 +6,16 @@ cfcFactions:RegisterDermaMenu("View News", Panel, 10)
...
-    self.NewsBox:SetMultiline(true)
+    self.NewsBox:Dock( FILL )
+    self.NewsBox:SetEditable( false )
+    self.NewsBox:SetMultiline( true )
 
-    self.NewsBox:SetText(cfcFactions.News)
-    string.Replace(self.NewsBox:GetText(), "%s", LocalPlayer():Nick())
-    net.Start("CFC_Fac_RequestNews")
+    self.NewsBox:SetText( cfcFactions.News )
+    string.Replace( self.NewsBox:GetText( ), "%s", LocalPlayer( ):Nick() )
```
Comment (sarahsturgeon) [inline lua/cfcfactions/core/cl/dermas/cl_newsderma.lua]:
```suggestion
    string.Replace( self.NewsBox:GetText(), "%s", LocalPlayer():Nick() )
```

File: lua/cfcfactions/core/cl/dermas/minis/cl_alertbox.lua [hunk 26]
```diff
@@ -9,24 +9,24 @@ local AlertInc = 0
 
 function Panel:Init()
     self:SetSize(250, 50)
-    if not type(msgtype) == "table" then msgtype = cfg.MsgType.Msg end
+    if not type( msgtype ) == "table" then msgtype = cfg.MsgType.Msg end
 
     self.MiniPanel = vgui.Create("DPanel", self)
-    self.MiniPanel:SetSize(self:GetWide(), self:GetTall())
-    self.MiniPanel:Dock(FILL)
+    self.MiniPanel:SetSize( self:GetWide( ), self:GetTall( ) )
```
Comment (sarahsturgeon) [inline lua/cfcfactions/core/cl/dermas/minis/cl_alertbox.lua]:
```suggestion
    self.MiniPanel:SetSize( self:GetWide(), self:GetTall() )
```

File: lua/cfcfactions/core/cl/dermas/minis/cl_alertbox.lua [hunk 27]
```diff
@@ -9,24 +9,24 @@ local AlertInc = 0
...
     self.err_icon:SetPos( 10, 35 )  -- Move it into frame
     self.err_icon:SetSize( 555, 150 )   
-    self.err_icon:Dock(LEFT)
+    self.err_icon:Dock( LEFT )
 
     self.ErrMsg = vgui.Create("DLabel", self.MiniPanel)
-    self.ErrMsg:Dock(RIGHT)
-    self.ErrMsg:SetColor(msgtype and msgtype ~= nil or Color(255,0,0))
+    self.ErrMsg:Dock( RIGHT )
+    self.ErrMsg:SetColor(msgtype and msgtype ~= nil or Color( 255,0,0) )
```
Comment (sarahsturgeon) [inline lua/cfcfactions/core/cl/dermas/minis/cl_alertbox.lua]:
```suggestion
    self.ErrMsg:SetColor( msgtype and msgtype ~= nil or Color( 255, 0, 0 ) )
```

File: lua/cfcfactions/core/sv/sv_factionmanagment.lua [hunk 28]
```diff
@@ -299,9 +299,9 @@ end
 concommand.Add("fpvp_testpermsys", factionTestPermSys)
 
 local function factionDebugTest(ply, cmd, args)
-    if fpm:IsDev(ply) then
-         cfcFactions:RemoveFaction(ply, ply:GetFactionID())
-         ply:concommand("fpvp_createfaction")
+    if fpm:IsDev( ply ) then
+         cfcFactions:RemoveFaction(ply, ply:GetFactionID( ) )
```
Comment (sarahsturgeon) [inline lua/cfcfactions/core/sv/sv_factionmanagment.lua]:
```suggestion
         cfcFactions:RemoveFaction( ply, ply:GetFactionID() )
```

File: lua/cfcfactions/core/sv/sv_factions.lua [hunk 29]
```diff
@@ -117,64 +117,64 @@ function cfcFactions:CreateFaction(owner, name, color, description, inviteOnly,
...
     return cfcFactions.Factions[TmpUnqID]
 end
 
 --Checks a specifc string to see if it is unique amongst other factions.
-function cfcFactions:isUniqueName(faction_name)
-    for k, v in pairs(cfcFactions.Factions) do
-        if string.lower(string.Trim(v.Name)) == string.lower(string.Trim(faction_name)) then
+function cfcFactions:isUniqueName( faction_name )
+    for k, v in pairs( cfcFactions.Factions ) do
+        if string.lower( string.Trim(v.Name) ) == string.lower( string.Trim(faction_name) ) then
```
Comment (sarahsturgeon) [inline lua/cfcfactions/core/sv/sv_factions.lua]:
```suggestion
        if string.lower( string.Trim( v.Name ) ) == string.lower( string.Trim( faction_name ) ) then
```

File: lua/cfcfactions/core/sv/sv_factions.lua [hunk 30]
```diff
@@ -218,25 +218,25 @@ function cfcFactions:EditFaction(id, name, color, description, inviteOnly, playe
...
             --TODO: Remove all players too
         end
     end
 end
 
 local function requestFactionNews(len, ply)
     --Look into a better way of sending faction news to client
-    for k, v in pairs(string.Explode("\n", cfcFactions:LoadNews())) do
-        net.Start("CFC_Fac_SendNews")
+    for k, v in pairs(string.Explode("\n", cfcFactions:LoadNews( )) ) do
```
Comment (sarahsturgeon) [inline lua/cfcfactions/core/sv/sv_factions.lua]:
```suggestion
    for k, v in pairs( string.Explode( "\n", cfcFactions:LoadNews() ) ) do
```

File: lua/cfcfactions/core/sv/sv_init.lua [hunk 31]
```diff
@@ -66,7 +66,7 @@ function cfcFactions:InitializeFactions()
     --load news
     cfcFactions:LoadNews()
 end
-hook.Add("Initialize", "cfcInitializeFactions", cfcFactions:InitializeFactions())
+hook.Add("Initialize", "cfcInitializeFactions", cfcFactions:InitializeFactions( ) )
```
Comment (sarahsturgeon) [inline lua/cfcfactions/core/sv/sv_init.lua]:
```suggestion
hook.Add( "Initialize", "cfcInitializeFactions", cfcFactions:InitializeFactions() )
```

File: lua/cfcfactions/core/sv/sv_init.lua [hunk 32]
```diff
@@ -83,25 +83,25 @@ end
...
 --InitialSpawn hook, fetches the data and properly sets it serverside
-local function cfcOnPlayerInitialSpawn(ply)
+local function cfcOnPlayerInitialSpawn( ply )
 
     --Always load a user as if never exsisted. Afterwards, load their proper data from source
     cfcFactions.Users:registeruser(ply, nil, nil)
 
-    for k, v in pairs(string.Explode("\n", cfcFactions:LoadNews())) do
-        net.Start("CFC_Fac_SendNews")
+    for k, v in pairs(string.Explode("\n", cfcFactions:LoadNews( )) ) do
```
Comment (sarahsturgeon) [inline lua/cfcfactions/core/sv/sv_init.lua]:
```suggestion
    for k, v in pairs( string.Explode( "\n", cfcFactions:LoadNews() ) ) do
```

File: lua/cfcfactions/core/sv/sv_init.lua [hunk 33]
```diff
@@ -83,25 +83,25 @@ end
...
 --PlayerSpawn hook
 --handles loading the user and properly setting their faction upon entering the server
-local function cfcPlayerSpawn(ply)
-    if not IsValid(ply) then return end
+local function cfcPlayerSpawn( ply )
+    if not IsValid( ply ) then return end
     ply:FetchUserData()
 end
-hook.Add("PlayerSpawn", "cfcPlayerSpawn", cfcPlayerSpawn(ply))
+hook.Add("PlayerSpawn", "cfcPlayerSpawn", cfcPlayerSpawn( ply) )
```
Comment (sarahsturgeon) [inline lua/cfcfactions/core/sv/sv_init.lua]:
```suggestion
hook.Add( "PlayerSpawn", "cfcPlayerSpawn", cfcPlayerSpawn( ply ) )
```

File: lua/cfcfactions/core/sv/sv_permsys.lua [hunk 34]
```diff
@@ -121,21 +121,21 @@ end
...
+        fpm:authUser( players )
     end
 end
 
 --Auths a user and allows them to use factions properly. If not, things make explode
 --Or simply just don't want them using it
-function fpm:authUser(player)
-    print("Authenticating user " .. player:SteamID())
+function fpm:authUser( player )
+    print("Authenticating user " .. player:SteamID( ) )
```
Comment (sarahsturgeon) [inline lua/cfcfactions/core/sv/sv_permsys.lua]:
```suggestion
    print( "Authenticating user " .. player:SteamID() )
```

File: lua/cfcfactions/core/sv/sv_permsys.lua [hunk 35]
```diff
@@ -184,8 +184,8 @@ end
 
 --Adds a permission to the player. True if success, false if otherwise
 function fpm:addPermission(player, permission)
-    if not IsValid(player) or not player:IsPlayer() then print("Unable to add permission, invalid player") return false end
-    if not fpm:IsValidPermission(permission) then 
+    if not IsValid( player ) or not player:IsPlayer() then print("Unable to add permission, invalid player") return false end
```
Comment (sarahsturgeon) [inline lua/cfcfactions/core/sv/sv_permsys.lua]:
```suggestion
    if not IsValid( player ) or not player:IsPlayer() then print( "Unable to add permission, invalid player" ) return false end
```

File: lua/cfcfactions/core/sv/sv_permsys.lua [hunk 36]
```diff
@@ -210,33 +210,33 @@ function fpm:revokePermission(player, permission)
...
 end
 
-function fpm:IsFactionAdmin(player)
+function fpm:IsFactionAdmin( player )
     if self:hasPermission(player, "IsFactionsAdmin") then return true end
     return false
 end
 
-hook.Add("Initialize", "cfcInitializeUsers", fpm:authAllUsers())
+hook.Add("Initialize", "cfcInitializeUsers", fpm:authAllUsers( ) )
```
Comment (sarahsturgeon) [inline lua/cfcfactions/core/sv/sv_permsys.lua]:
```suggestion
hook.Add( "Initialize", "cfcInitializeUsers", fpm:authAllUsers() )
```

**Review by sarahsturgeon — COMMENTED** [review]
Fixes

File: lua/cfcfactions/core/cl/dermas/cl_mainderma.lua [hunk 37]
```diff
@@ -81,85 +81,85 @@ end
...
     self.closeButton = vgui.Create('DButton', self)
-    self.closeButton:SetFont('CFC_Normal')
-    self.closeButton:SetText('[X]')
+    self.closeButton:SetFont( 'CFC_Normal' )
+    self.closeButton:SetText( '[X]' )
     self.closeButton.Paint = function() end
     self.closeButton:SetColor(Color(255, 255, 255))
     self.closeButton:SetSize(32, 32)
-    self.closeButton:SetPos(self:GetWide() - 35, 5)
+    self.closeButton:SetPos( self:GetWide( ) - 35, 5)
```
Comment (sarahsturgeon) [inline lua/cfcfactions/core/cl/dermas/cl_mainderma.lua]:
```suggestion
    self.closeButton:SetPos( self:GetWide() - 35, 5 )
```

File: lua/cfcfactions/core/sh/sh_sharedutils.lua [hunk 38]
```diff
@@ -75,12 +75,12 @@ cfcFactions.ErrorMessages = {
 }
 
 function cfcFactions:TimeStamp()
-    return os.date( "%H:%M:%S - %d/%m/%Y", os.time())
+    return os.date( "%H:%M:%S - %d/%m/%Y", os.time( ) )
```
Comment (justinmuskopf) [inline lua/cfcfactions/core/sh/sh_sharedutils.lua]:
```suggestion
    return os.date( "%H:%M:%S - %d/%m/%Y", os.time() )
```

File: lua/cfcfactions/core/sv/sv_users.lua [hunk 39]
```diff
@@ -41,29 +41,29 @@ function cfcuser:registeruser(user, factionid, rank)
...
-function cfcuser:UserExists(user)
+function cfcuser:UserExists( user )
 
 	if not user:IsPlayer() then 
 		--Error out, not a player
 		return 
 	end
 
-	if table.HasValue(cfcuser, user:SteamID64()) then
+	if table.HasValue(cfcuser, user:SteamID64( ) ) then
```
Comment (justinmuskopf) [inline lua/cfcfactions/core/sv/sv_users.lua:51]:
```suggestion
	if table.HasValue(cfcuser, user:SteamID64() ) then
```

**Review by justinmuskopf — CHANGES_REQUESTED** [review]
pls

**Comment by sarahsturgeon** [comment]
Closing this in favor of https://github.com/brandonsturgeon/cfc_factions_3/pull/46

( :( )

**Pull request closed without merging**